src/compas_timber/connections/solver.py

```diff
@@ -372,3 +371,55 @@ def does_segment_intersect_outline(segment, polyline, tol=TOL):
         if intersection_segment_polyline(segment, polyline, tol.absolute)[0]:
             return True
         return is_point_in_polyline(segment.point_at(0.5), polyline, in_plane=False, tol=tol)
+
+class BeamConnectionTopologyResults(Data):
```

Comment:
i find this a bit long and unspecific, what do you think about something like `BeamSolverResult` (singular)?